### DISCUSSION TITLE
GP11 Black shuts off after scanning QR Codes

### DISCUSSION THREAD

**Discussion by scubabs** [gopro/labs#594] [Q&A]
Have a new GP 11 Black and installed the Lab for scanning QR codes. When I scan a QR code it changes my settings and about 10 seconds later it shuts the gopro off. And after I turn it back on the camera stays on for only a few seconds and shuts off. Anyone know how to fix this?

**Comment by dnewman-gpsw** [comment]
All depends on what you scanned.  You can try scanning !RESET, but you will may need to share your log file to reveal the error. Log file is in MISC/qrlog.txt

**Reply by scubabs** [reply]
I scanned the ones created by the 4k Pro ( https://www.bluewaterphotostore.com/gopro-settings-qr-codes ) and ones I created from the QRControl App.

**Reply by dnewman-gpsw** [reply]
That is not enough information. Need you logs files or and complete list of the commands you scanned.

Scan this:  
![image](https://github.com/gopro/labs/assets/10856084/557cc988-ff43-4432-944e-76211a3bfbb1)

Does the problem go away?
